src/ui/Base/ComponentOptions.ts

```diff
@@ -920,13 +920,14 @@ export class ComponentOptions {
   }
 
   static isElementScrollable(element: HTMLElement) {
-    return $$(element).css('overflow-y') == 'scroll' || element.style.overflowY == 'scroll';
+    const overflowProperty = $$(element).css('overflow-y') || element.style.overflowY;
```

Comment:
Why check using `$$` with a fallback to a DOM method? Why not use just one of the two, preferably the DOM method?